lib/rules/call-indentation.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Check indentation at the beginning and end of a function call
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+   var sourceCode = context.getSourceCode(),
+       lines = sourceCode.getLines();
+
+   //--------------------------------------------------------------------------
+   // Helpers
+   //--------------------------------------------------------------------------
+
+   function lineIndent(lineNumber) {
+      var line = lines[lineNumber - 1];
+
+      if (line === undefined) {
+         throw new Error('Invalid lineNumber ' + lineNumber + ' supplied to lineIndent');
+      }
+
+      return line[0] === ' ' ? line.match(/[^ ]/).index : 0;
+   }
+
+   function validateCallIndentation(node) {
+      var firstLineIndent = lineIndent(node.callee.loc.end.line),
+          lastLineIndent = lineIndent(node.loc.end.line);
+
+      if (firstLineIndent !== lastLineIndent) {
+         context.report({
+            node: node,
+            message: 'Call expressions must begin and end with the same indentation',
+         });
+      }
+   }
+
+   //--------------------------------------------------------------------------
+   // Public
+   //--------------------------------------------------------------------------
+
+   return {
+      'CallExpression': function(node) {
+            // validate multi-line function indentation
```

Comment:
this whole block is indented wrong ....

![](https://i.ytimg.com/vi/K5tQn9f2Qc0/hqdefault.jpg)